lib/fizzy/capi.cpp

```diff
@@ -11,6 +11,35 @@
 
 namespace
 {
+inline const FizzyValueType* wrap(const fizzy::ValType* value_type) noexcept
+{
+    return reinterpret_cast<const FizzyValueType*>(value_type);
+}
+
+inline fizzy::ValType unwrap(FizzyValueType value_type) noexcept
+{
+    return static_cast<fizzy::ValType>(static_cast<uint8_t>(value_type));
+}
+
+inline FizzyFunctionType wrap(const fizzy::FuncType& type) noexcept
+{
+    return {wrap(&type.inputs[0]), type.inputs.size(), wrap(&type.outputs[0]), type.outputs.size()};
+}
+
+inline fizzy::FuncType unwrap(const FizzyFunctionType& type)
+{
+    fizzy::FuncType func_type{std::vector<fizzy::ValType>(type.inputs_size),
```

Comment:
We could optimize it later on the C++ side to not require vector allocation to pass function type (i.e. replace `vector` with `span` in `fizzy::FuncType`)